docker-compose.yml

```diff
@@ -35,14 +35,16 @@ x-sentry-defaults: &sentry_defaults
     PYTHONUSERBASE: "/data/custom-packages"
     SENTRY_CONF: "/etc/sentry"
     SNUBA: "http://snuba-api:1218"
+    REQUESTS_CA_BUNDLE: "/etc/ssl/certs/ca-certificates.crt"
+    GRPC_DEFAULT_SSL_ROOTS_FILE_PATH_ENV_VAR: "/etc/ssl/certs/ca-certificates.crt"
     # Leaving the value empty to just pass whatever is set
     # on the host system (or in the .env file)
     SENTRY_EVENT_RETENTION_DAYS:
   volumes:
     - "sentry-data:/data"
     - "./sentry:/etc/sentry"
     - "./geoip:/geoip:ro"
-    - "./certificates:/usr/local/share/ca-certificates:ro"
+    - "./certificates:/usr/local/share/ca-certificates"
```

Comment:
Ah! Permissions after all. :) This is the heart of the fix, ya?

Reply:
Not really, trying to bring this back TBH. The issue was using `$dc exec -T web` instead of `$dcr web`.

Reply:
Interesting. Say more? Why did that cause the custom certificate to not be appended to `/ets/ssl/certs/ca-certificates.crt`?

Reply:
`docker-compose exec` runs the command in the same container that is created by `docker-compose up`. From the logs, I deduced that `update-ca-certificates` were running successfully for all Sentry containers (including the derived `cleanup` cron containers) except for `web`. On `web`, it was getting cut off at the point where it was scanning the disk for new certs. Then I realized we were running `docker-compose exec <our test script>` too early, before the container fully starts up, and interrupt the certificate update step.

Reply:
Okay, and for the record `$dcr` [expands to](https://github.com/getsentry/onpremise/blob/3f713c27439e8e97cefb5d0a41e3e0ad99a3c0e6/install/_lib.sh#L29) `docker run --rm`, so we're running the tests in an isolated new container vs. running in the existing one. Makes sense, thanks! 👍 

https://github.com/getsentry/onpremise/blob/3f713c27439e8e97cefb5d0a41e3e0ad99a3c0e6/_integration-test/run.sh#L126